sycl/test-e2e/LLVMIntrinsicLowering/bitreverse.cpp

```diff
@@ -0,0 +1,189 @@
+// Test that llvm.bitreverse is lowered correctly by llvm-spirv
```

Comment:
Can this test run on all hardware including HIP/CUDA? If not, we need to restrict it using `REQUIRES` or `UNSUPPORTED`

Reply:
HIP/CUDA is now marked as unsupported.